grammars/scss.cson

```diff
@@ -291,27 +291,34 @@
   'at_rule_include':
     'patterns': [
       {
-        'begin': '\\s*((@)include\\b)\\s*'
+        'name': 'meta.at-rule.include.scss'
```

Comment:
We're currently discussing how to make the grammars the most readable in atom/language-php#137.  If you have any suggestions you should definitely voice them over there as well.